src/e2e/resources/data/AdminAccountsPageE2ETest.json

```diff
@@ -96,6 +102,29 @@
         "sectionLevel": {},
         "sessionLevel": {}
       }
+    },
+    "AAccounts.instr3-AAccounts.CS2103": {
+      "googleId": "tm.e2e.AAccounts.instr3",
+      "courseId": "tm.e2e.AAccounts.CS2103",
+      "name": "Teammates Instr3",
+      "email": "AAccounts.instr3@gmail.tmt",
+      "role": "Co-owner",
+      "isDisplayedToStudents": true,
```

Comment:
So from what I gathered about E2E tests and the "happy path," I figured a test should check the admin can successfully delete an instructor, as deleting a last instructor of a course is more of an edge case. Thus, it required another instructor to be created

Reply:
It's the AdminAccountsPageE2ETest. I figured I should fix the issue, and all associated unit tests and E2E tests, in one PR. The E2E test was set up to create a course with one invisible instructor, then simulate the admin deleting the instructor. Because this behavior was changed with this PR, I modified the initial parameters.

Reply:
I see, so the changes you have made would cause AdminAccountsPageE2ETest to fail as the there is currently 1 instructor attached to the course CS2103 (they are invisible as well, causing the test to fail too).

Thanks for clarifying, this makes sense to me